cmd/initialize.go

```diff
@@ -25,9 +25,10 @@ var initializeCmd = &cobra.Command{
 
 func initializeDatabase() {
 	runDatabaseEffects(func(db *gorm.DB) {
-		if db.Migrator().CreateTable(database.Tables...) != nil {
+		if db.AutoMigrate(database.Tables...) != nil {
```

Comment:
let's keep `initialize` as is and create a new command for `migrate`

Reply:
okay.  new commit pushed

Reply:
`migrate` can be run for both schema creation and migration.  it is a valid option to not run `initialize`.